src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DraftReferralTest.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
+import java.util.Date
+import java.util.UUID
+
+class DraftReferralTest : IntegrationTestBase() {
+
+  @Test
+  fun `create draft referral dto from empty referral`() {
+    val referral = Referral()
+    var draftReferral = DraftReferral(referral)
+
+    assertThat(draftReferral.id).isNull()
+    assertThat(draftReferral.created).isNull()
+  }
+
+  @Test
+  fun `create draft referral dto from referral`() {
+    val referral = Referral()
+    referral.created = Date()
+    referral.id = UUID.randomUUID()
+    var draftReferral = DraftReferral(referral)
+
+    assertThat(draftReferral.id).isEqualTo(referral.id.toString())
```

Comment:
i really strive to avoid replicating the logic under test and instead test against concrete values e.g. 

```
    val date = SimpleDateFormat("yyyy-MM-dd HH:mm:ss").parse("2020-12-04 10:42:43")
    val id = UUID.fromString("70d3a47c-d539-4f76-8fc9-1c50e34aea29")
    var draftReferral = DraftReferral(Referral(created=date, id=id))

    assertThat(draftReferral.id).isEqualTo("70d3a47c-d539-4f76-8fc9-1c50e34aea29")
    assertThat(draftReferral.created).isEqualTo("2020-12-04T10:42:43Z")
```

see https://testing.googleblog.com/2014/07/testing-on-toilet-dont-put-logic-in.html for more on this